walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -30,4 +30,22 @@ public void testGreeting() {
 
     Assert.assertEquals("Hello Ada", greeting);
   }
+	@Test
+	public void testGreetingTrimsWhitespace() {
+	Greeter greeter = new Greeter();
+
+	String greeting = greeter.greet("   Ada   ");
+
+	// Whitespace should be trimmed
+	Assert.assertEquals("Hello Ada", greeting);
+	}
+	@Test
+	public void testGreetingOddCharsWithWhitespace() {
+	Greeter greeter = new Greeter();
+
+	String greeting = greeter.greet("   Ross$$$");
+
+	// Whitespace should be trimmed
+	Assert.assertEquals("Hello Ross", greeting);
+	}
```

Comment:
Formatting is off.

Testing is broken into three parts:

Variable instantiation

Execution of unit test

Assertion data is correct.

In last two steps, do not inline testing data, keep it with the 1st section and call upon the variable.

Reply:
I was just following the format they gave